README.md

```diff
@@ -159,6 +159,34 @@ platform](https://github.com/asreview/systematic-review-datasets).
 asreview data dedup benchmark:van_de_schoot_2017 -o van_de_schoot_2017_dedup.csv
 ```
 
+### Data Stack (Experimental)
+Combine as many datasets as you want into a single dataset.
+
+❗ Stack is an experimental feature. We would love to hear your feedback. Please keep in mind that this feature can change in the future.
+
+#### Data format
+Your data files need to be in tabular file format or RIS file format. All input files should be in the same format, the output path should also be of the same file format.
+
+- **Tabular file format:**
+Supported tabular file formats are `.csv`, `.tab`, `.tsv` or `.xlsx`.
+
+
+- **RIS file format:**
+RIS file formats have `.ris` or `.txt` as extension and are used by digital libraries.
+Read [how to format](https://asreview.readthedocs.io/en/latest/data_format.html) your RIS files.
+
+#### Run script
```

Comment:
```suggestion
```

Stay close to the format used in the other sections. If you disagree with the format, open another PR and change everything at once.